apps/system-e2e/src/support/session.ts

```diff
@@ -147,3 +147,16 @@ export async function session({
   await sessionValidationPage.close()
   return context
 }
+
+export async function judicialSystemSession({ browser }: { browser: Browser }) {
+  const context = await browser.newContext()
+  const page = await context.newPage()
+  const authUrlPrefix = urls.authUrl
+  await ensureCognitoSessionIfNeeded(
+    page,
+    JUDICIAL_SYSTEM_HOME_URL,
+    authUrlPrefix,
+  )
+  await page.close()
+  return context
+}
```

Comment:
Why is this needed? Can't you use this in your test?
```
session({
  browser,
  homeUrl: '/api/auth/login?nationalId=9999999999',
  phoneNumber: null,
  authUrl: '/',
  idsLoginOn: false,
  delegation: false,
})
```

Reply:
We use a way to bypass the login that's different than in other projects. What we do is when we go to `localhost/api/auth/login?nationalId=some_national_id` we bypass the login and are redirected to our homepage, i.e. `/krofur`.

`session` checks if `homeurl` is correct and that doesn't happen in our case, because we use this bypass mechanism. 

<img width="539" alt="Screenshot 2023-03-08 at 13 57 16" src="https://user-images.githubusercontent.com/3789875/223731663-31b2f169-9ea3-4320-838e-a880ca5a671c.png">

This was created after a talk with @pshomov, who suggested this.

<img width="1229" alt="Screenshot 2023-03-08 at 13 58 11" src="https://user-images.githubusercontent.com/3789875/223731979-29f28cf8-3b19-4adf-bf6e-dab3c99c94b3.png">

Reply:
I don't know whether I agree. I would prefer you used our session mechanism, and instead updated it to allow your workflow. Surely, we will have to test other services in the future with similar requirements, which would benefit from a common `session` mechanism.

This would also reduce maintenance :crossed_fingers: